tensorflow_datasets/audio/commonvoice.py

```diff
@@ -0,0 +1,71 @@
+"""
+Mozilla Common Voice Dataset (English)
+"""
+import os
+import csv
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+_DOWNLOAD_LINK = {
+    "en": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/en.tar.gz"}
+_SPLITS = {
+    tfds.Split.TRAIN: "train",
+    tfds.Split.TEST: "test",
+    tfds.Split.VALIDATION: "validated"}
+
+
+class CommonVoice(tfds.core.GeneratorBasedBuilder):
+    """
```

Comment:
Let's use ["heavy" configuration](https://www.tensorflow.org/datasets/add_dataset#dataset_configuration) for the different languages, so that eventually we could have all of them. You can start with just 1 for English.

Reply:
Looking into it.

Reply:
@rsepassi  Do you mean something like this?
[https://github.com/tensorflow/datasets/blob/dde9c9d7fd14fff3458e59c120f50416657633a7/tensorflow_datasets/audio/librispeech.py#L62](https://github.com/tensorflow/datasets/blob/dde9c9d7fd14fff3458e59c120f50416657633a7/tensorflow_datasets/audio/librispeech.py#L62)

Reply:
Yes.